legal-api/Makefile

```diff
@@ -115,16 +115,19 @@ push: #build ## Push the docker container to the registry & tag latest
     docker tag $(DOCKER_NAME) $(REGISTRY_IMAGE):latest ;\
     docker push $(REGISTRY_IMAGE):latest
 
-VAULTS=`cat devops/vaults.json`
-update-env: ## Update env from 1pass
-	oc -n "$(OPS_REPOSITORY)-$(TAG_NAME)" exec "dc/vault-service-$(TAG_NAME)" -- ./scripts/1pass.sh \
-		-m "secret" \
-		-e "$(TAG_NAME)" \
-		-a "$(DOCKER_NAME)-$(TAG_NAME)" \
-		-n "$(OPENSHIFT_REPOSITORY)-$(TAG_NAME)" \
-		-v "$(VAULTS)" \
-		-r "true" \
-		-f "false"
+
+# 1Password CLI1 will be deprecated on Oct 1, 2024
```

Comment:
Only in legal-api? what about rest of the makefile in lear?

Reply:
Think it's ok to verify the changes for one component and then modify the others. How do you think?

Reply:
you need to remove these call
![image](https://github.com/user-attachments/assets/27d148a5-b2b9-4f97-88db-f9529be3ab85)

Reply:
> Think it's ok to verify the changes for one component and then modify the others. How do you think?

I am ok with it

Reply:
@kzdev420  FYI, in case you didn't see Vysakh's comment

Reply:
Hi @chenhongjing @vysakh-menon-aot 

Updated.